rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly make assign the `attr_accessor` to the `book` and `person` instance variables here.

Reply:
Hi @TedLivist, thanks for your review. The required changes were applied.